openedx/core/djangoapps/user_authn/views/register.py

```diff
@@ -522,6 +526,16 @@ def post(self, request):
         if response:
             return response
 
+        try:
+            trigger_filter("openedx.lms.auth.pre_register.filter.v1", data)
```

Comment:
it should be `data = trigger_filter("openedx.lms.auth.pre_register.filter.v1", data)` right?

Reply:
We can do something like this.

out = trigger_filter(..)
data = out.get('data')

But in any case, I think that the changes to 'data' in the functions are already being stored in the data dictionary passed as parameter

Reply:
Allright, `out = trigger_filter(..)` makes more sense.

Now, about the data being pass as reference, this could work, but we should not count on this. A filter could replace the dict with a new one and we would completely ignore that.